subgraph/origin/schema.graphql

```diff
@@ -0,0 +1,9 @@
+type AvailableStateRoot @entity {
+  id: ID!
+  blockHeight: BigInt! # uint256
+  stateRoot: Bytes! # bytes32
```

Comment:
I feel we doesn't need this.

Reply:
Done!
We should also remove this from event also. cc: @deepesh-kn @jayeshbairagi